### PR TITLE
NAS-107708 / 12.0 / Nas 107708

### PR DESCRIPTION
**Pull request by dszidi**
Add support for ES24f and ES102
Fix orientation for E60 and ES60 (They should be ordered from top to bottom not left to right)

### CODE REVIEW

File: src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks.component.ts [hunk 1]
```diff
@@ -15,8 +15,10 @@ import { ES12 } from 'app/core/classes/hardware/es12';
 import { E16 } from 'app/core/classes/hardware/e16';
 import { E24 } from 'app/core/classes/hardware/e24';
 import { ES24 } from 'app/core/classes/hardware/es24';
+import { ES24f } from 'app/core/classes/hardware/es24f';
```
Comment (darkfiberiru) [inline src/app/pages/system/viewenclosure/enclosure-disks/enclosure-disks.component.ts:18]: Should be ES24F

**Review by darkfiberiru — CHANGES_REQUESTED** [review]
Should be s/ES24f/ES24F/g every where. Original ticket had type that Andrew Nguyen Product PM informed me of.

**Review by dkmullen — APPROVED** [review]
Looks good - there is one active console log on line 496 in enclosure-disks.compoment

**Comment by dkmullen** [comment]
@darkfiberiru Could you look at this PR again and see if your requested changes are done?

**Review by darkfiberiru — APPROVED** [review]
My suggestion look fixed. Good to go. Haven't tested on real system yet but can once we have a nightly

**Pull request merged**